nextjs-app/app/program-rules/page.tsx

```diff
@@ -3,6 +3,11 @@ import MarqueeTitle from "@/components/common/MarqueeTitle";
 import Benefits from "@/components/pages/program-rules/Benefits";
 import JoinUs from "@/components/pages/program-rules/JoinUs";
 import GroupsIntro from "@/components/pages/program-rules/GroupsIntro";
+import Captain from "@/components/pages/program-rules/Captain";
+import PositiveFeedback from "@/components/pages/program-rules/PositiveFeedback";
```

Comment:
```suggestion
import Captains from "@/components/pages/program-rules/Captain";
import PositiveFeedbacks from "@/components/pages/program-rules/PositiveFeedback";
```

I am thinking about changing these names to plural name to be consistent with `Reminders`

Reply:
[17a4360](https://github.com/thementorship-tw/thementorship.tw/pull/107/commits/17a4360981eed2fd1ad1ef601578fb8cc93f8163)
Rename Captain and PositiveFeedback components to plural form~